src/catalog/catalog.cpp

```diff
@@ -373,16 +374,22 @@ void Catalog::dropIndex(Transaction* transaction, table_id_t tableID,
     indexes->dropEntry(transaction, uniqueName, entry->getOID());
 }
 
-bool Catalog::containsFunction(const Transaction* transaction, const std::string& name) const {
-    return functions->containsEntry(transaction, name);
+bool Catalog::containsFunction(const Transaction* transaction, const std::string& name,
+    bool useInternal) const {
```

Comment:
I had this question about useInternal for a while.

If useInternal is true, should we still check external entries? The current semantic is to always check external entries.

Reply:
Yeah we should still check external. `useInternal` is not restricting the visibility to internal only.